crates/holochain/src/conductor/conductor.rs

```diff
@@ -1949,6 +2001,48 @@ mod app_status_impls {
                 .await?;
             Ok(disabled_app)
         }
+
+        /// Get from [`AppManifest`] the [`P2pConfigOverrides`] to apply when creating cells for the app.
+        pub(crate) fn p2p_config_overrides(manifest: &AppManifest) -> Option<P2pConfigOverrides> {
+            let mut overrides = P2pConfigOverrides::default();
+            match manifest {
+                AppManifest::V0(manifest) => {
+                    overrides.bootstrap_url = manifest.bootstrap_url.clone();
+                    #[cfg(any(
+                        feature = "transport-tx5-backend-libdatachannel",
+                        feature = "transport-tx5-backend-go-pion",
+                        feature = "transport-tx5-datachannel-vendored"
+                    ))]
+                    {
+                        overrides.signal_url = manifest.signal_url.clone();
+                    }
+                }
+            }
+            if overrides.is_overriding() {
+                Some(overrides)
+            } else {
+                None
+            }
+        }
+
+        /// Check whether there is any already installed app with conflicting P2P overrides.
+        ///
+        /// If there is a conflicting app, return its [`CellId`].
```

Comment:
It actually might be most helpful to have both the CellId and app id as context for errors in this case but that's a minor thing.